hourly_template.html

```diff
@@ -14,12 +14,13 @@ <h3>Hourly Job Status: {{runtime_long}}</h3>
 
 
     <table border=1>
-      <tr><th>File</th> <th>Status</th> <th>Age (Hours)</th><th>MaxAge</th> <th>Last Date</th> </tr>
+      <tr><th>Task</th><th>File</th> <th>Status</th> <th>Age (Hours)</th><th>MaxAge</th> <th>Last Date</th> </tr>
```

Comment:
I added "task" here basically to have something that worked as a comment.  Now we are checking the "arc" copy of the snapshot and the mta copy...  However it does give us "arc" for just about everything else in the table that is not helpful.

![table](https://user-images.githubusercontent.com/791508/34122693-4a95c000-e3fb-11e7-9f68-139fed8f63a3.png)